Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -271,6 +271,21 @@ public class MXRAndroidSystem : IMXRSystem {
                 Debug.unityLogger.Log(LogType.Warning, TAG, "ConnectToWifiNetwork ignored. System is not available (not bound to messenger.");
         }
 
+        public void ConnectToEnterpriseWifiNetwork(EnterpriseWifiConnectionRequest enterpriseWifiConnectionRequest) {
+
+            if(enterpriseWifiConnectionRequest == null)
+                throw new ArgumentNullException(nameof(enterpriseWifiConnectionRequest));
+
+            if (messenger.IsBoundToService) {
+                if (LoggingEnabled)
+                    Debug.unityLogger.Log(LogType.Log, TAG, "ConnectToEnterpriseWifiNetwork called. Invoking over JNI: connectToEnterpriseWifiNetworkAsync");
+                string json = JsonConvert.SerializeObject(enterpriseWifiConnectionRequest, Formatting.None, new StringEnumConverter());
```

Comment:
We log by default, and this would log passwords - I think we cannot check this in.

Reply:
We don't log JSON here, so this is good to go.